app/services/pipeline_report_service.rb

```diff
@@ -137,6 +146,48 @@ def fetch_taxon_counts(_pipeline_run_id, _background_id)
     return taxon_counts_and_summaries_query.pluck(*FIELDS_TO_PLUCK)
   end
 
+  def zero_metrics(tax_id, tax_level, count_type, mean, stdev)
+    # Fill in default zero values for FIELDS_TO_PLUCK.
+    # Necessary for taxons absent from sample to match the taxon_counts_and_summaries structure,
+    # since they're fetched from TaxonSummary, which doesn't have some columns listed in FIELDS_TO_PLUCK.
+    # The two nil values (genus_taxid and name) will be filled in when counts_by_tax_level is
+    # transformed to be indexed by taxid and count type.
+    [
```

Comment:
This format is hard to maintain and understand. If you had fields to pluck it is hard to keep track of the values here.
Have you consider adding a FIELDS_DEFAULTS to include defaults for plucked fields e.g.
```
FIELDS_TO_PLUCK = {
    genus_taxid => nil,
    count => 0,
    percent_identity => 0,
	...
]
```
Then you can build this dynamically using this hash the FIELDS_TO_PLUCK.
End goal is to make it more obvious when adding fields and make it more readable.